documentation/src/docs/metrics.md

```diff
@@ -7,57 +7,50 @@ You may obtain a copy of the License at
+- startWithoutExporting(): Initializes SimpleMeterRegistry that holds the latest value of each meter in memory and does not export the data anywhere, typically for unit tests.
 - close(): Shutdown of Metrics service.
 - createStatsLogger(): Creates and returns a StatsLogger instance, which is used to retrieve a metric and do metric insertion and collection in Pravega code. 
 - createDynamicLogger(): Create a dynamic logger.
 
-## 1.2. Metric Logger — interface StatsLogger
-Using this interface we can register required metrics for simple types like [Counter](http://metrics.dropwizard.io/3.1.0/manual/core/#counters) and [Gauge](http://metrics.dropwizard.io/3.1.0/manual/core/#gauges) and some complex statistics type of Metric OpStatsLogger, through which we provide [Timer](http://metrics.dropwizard.io/3.1.0/manual/core/#timers) and [Histogram](http://metrics.dropwizard.io/3.1.0/manual/core/#histograms).
-```java
-public interface StatsLogger {
-    OpStatsLogger createStats(String name);
-    Counter createCounter(String name);
-    Meter createMeter(String name);
-    <T extends Number> Gauge registerGauge(String name, Supplier<T> value);
-    StatsLogger createScopeLogger(String scope);
-}
-```
+## 1.2. Metric Logger — interface StatsLogger 
+Using this interface we can register required metrics for simple types like 
+[Counter](https://micrometer.io/docs/concepts#_counters) and 
+[Gauge](https://micrometer.io/docs/concepts#_gauges) 
+and some complex statistics type of Metric OpStatsLogger, through which we provide 
+[Timer](https://micrometer.io/docs/concepts#_timers) and 
+[Distribution Summary](https://micrometer.io/docs/concepts#_distribution_summaries).
 
-- createStats(): Register and get a OpStatsLogger, which is used for complex type of metrics.
+[public interface StatsLogger](https://github.com/pravega/pravega/blob/master/shared/metrics/src/main/java/io/pravega/shared/metrics/StatsLogger.java)
+- createStats(): Register and get a OpStatsLogger, which is used for complex type of metrics. Notice the optional metric tags.
 - createCounter(): Register and get a Counter metric.
 - createMeter(): Create and register a Meter metric.
 - registerGauge(): Register a Gauge metric.
 - createScopeLogger(): Create the stats logger under given scope name.
 
 ### 1.3. Metric Sub Logger — OpStatsLogger
-OpStatsLogger provides complex statistics type of Metric, usually it is used in operations such as CreateSegment, ReadSegment, we could use it to record the number of operation, time/duration of each operation.
-```java
-public interface OpStatsLogger {
-    void reportSuccessEvent(Duration duration);
-    void reportFailEvent(Duration duration);
-    void reportSuccessValue(long value);
-    void reportFailValue(long value);
-    OpStatsData toOpStatsData();
-    void clear();
-}
-```
+OpStatsLogger provides complex statistics type of Metric, usually it is used in operations such as CreateSegment, 
```

Comment:
By "complex statistics" you mean a "histogram", right? If so, you may want to use the concept of histogram that looks more concrete. You can say that if the user is interested on measuring the latency of operations (`CreateSegment`, `ReadSegment`,...) this is the metric type to choose.

Reply:
Accepted suggestion